posthog/management/commands/compare_hogql_insights.py

```diff
@@ -0,0 +1,55 @@
+from django.core.management.base import BaseCommand
+
+from posthog.schema import HogQLQueryModifiers, MaterializationMode
+
+
+class Command(BaseCommand):
+    help = "Test if HogQL insights match their legacy counterparts"
+
+    def handle(self, *args, **options):
+        from posthog.models import Insight, Filter
+        from posthog.queries.trends.trends import Trends
+        from posthog.hogql_queries.legacy_compatibility.filter_to_query import filter_to_query
+        from posthog.hogql_queries.query_runner import get_query_runner
+
+        insights = (
+            Insight.objects.filter(filters__contains={"insight": "LIFECYCLE"}, saved=True, deleted=False, team_id=2)
```

Comment:
Why team 2? For me it's 1.

Reply:
That's what we have in production on the US cloud. The next version of this query should just check for all teams.